packages/cozy-harvest-lib/src/services/softDeleteOrRestoreAccounts.js

```diff
@@ -0,0 +1,85 @@
+import logger from './logger'
+import {
+  decryptString,
+  getOrganizationKey,
+  fetchAccountsForCipherId,
+  fetchKonnectorFromAccount,
+  fetchTriggersFromAccount,
+  updateAccountsAuth
+} from './utils'
+import get from 'lodash/get'
+import { ensureTrigger } from '../connections/triggers'
+import { translate as t } from 'cozy-ui/transpiled/react/I18n'
+
+const makeDecrypt = (vaultClient, orgKey) => encryptedVal =>
+  decryptString(encryptedVal, vaultClient, orgKey)
+
+const softDeleteOrRestoreAccounts = async (
```

Comment:
Can you add a small doc explaining the flow (maybe copy/pasting the description of this PR here) ? The "updateAccountsFromCipher" bit, restoring credentials, was a bit unclear to me, and could be a little bit clearer I think : "updateAccountsFromCipher will restore the credentials from the linked cipher" ?